dev-util/intel-graphics-compiler/intel-graphics-compiler-1.0.8173.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+PYTHON_COMPAT=( python{3_9,3_10} )
+LLVM_MAX_SLOT=10
+
+inherit cmake-multilib flag-o-matic llvm python-any-r1
```

Comment:
If multilib doesn't work, we should drop it.

Reply:
Yes, @thesamesam was faster :-)

Reply:
FWICS, only intel-neo depends on intel-graphics-compiler, which does not support multilib anyway, so it should be safe to drop from intel-graphics-compiler.

Reply:
Should now be 64bit only